xtable-utilities/src/test/java/org/apache/xtable/utilities/TestRunCatalogSync.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+package org.apache.xtable.utilities;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import lombok.SneakyThrows;
+
+import org.junit.jupiter.api.Test;
+
+class TestRunCatalogSync {
+
+  @SneakyThrows
+  @Test
+  void testMain() {
+    String catalogConfigYamlPath =
+        TestRunCatalogSync.class.getClassLoader().getResource("catalogConfig.yaml").getPath();
+    String[] args = {"-catalogConfig", catalogConfigYamlPath};
+    // Ensure yaml gets parsed and no op-sync implemented in TestCatalogImpl is called.
+    assertDoesNotThrow(() -> RunCatalogSync.main(args));
```

Comment:
Can we add? Can we add some state in the TestCatalogImpl we can check to see that it is called or whether the table metadata is synced?

Reply:
I will address this in few hours.

Reply:
I have a similar test setup in my continuous conversion PR if you want to take any of the setup code: https://github.com/apache/incubator-xtable/pull/593/files#diff-5e3d12ea3cafe27a7e24f5600786589eec233e518edb3a54ecf7a785c5aa335d

Reply:
There's some jar issue for test dependencies, will fix it tonight.